ntbs-integration-tests/NotificationPages/ManualTestResultEditPages.cs

```diff
@@ -0,0 +1,167 @@
+        {
+        }
+
+        public static IList<Notification> GetSeedingNotifications()
+        {
+            return new List<Notification>()
+            {
+                new Notification()
+                {
+                    NotificationId = Utilities.NOTIFICATION_WITH_MANUAL_TESTS,
+                    NotificationStatus = NotificationStatus.Notified,
+                    TestData = { ManualTestResults = new List<ManualTestResult> () { new ManualTestResult
+                    {
+                        ManualTestResultId = TEST_ID,
+                        TestDate = new DateTime(),
+                        ManualTestTypeId = (int)ManualTestTypeId.Smear,
+                        SampleTypeId = (int)SampleTypeId.LungBronchialTreeTissue,
+                        Result = Result.Positive,
+                    }}}
+                }
+            };
+        }
+
+        [Fact]
+        public async Task PostNewManualTestResult_ReturnsSuccessAndAddsResultToTable()
+        {
+            // Arrange
+            var NotificationId = Utilities.DRAFT_ID;
+            var url = GetCurrentPathForId(NotificationId);
+            var initialPage = await Client.GetAsync(url);
+            var initialDocument = await GetDocumentAsync(initialPage);
+
+
+            // Act
+            var formData = new Dictionary<string, string>
+            {
+                ["FormattedTestDate.Day"] = "2",
+                ["FormattedTestDate.Month"] = "2",
+                ["FormattedTestDate.Year"] = "2012",
+                ["TestResultForEdit.ManualTestTypeId"] = ((int)ManualTestTypeId.Smear).ToString(),
+                ["TestResultForEdit.SampleTypeId"] = ((int)SampleTypeId.Blood).ToString(),
+                ["TestResultForEdit.Result"] = ((int)Result.Negative).ToString(),
+            };
+            var result = await SendPostFormWithData(initialDocument, formData, url);
+
+            // Assert
+            result.AssertRedirectTo(GetPathForId(NotificationSubPaths.EditTestResults, NotificationId));
+            var testsListPage = await Client.GetAsync(GetRedirectLocation(result)); // Follow the redirect to see results table
+            var testsListDocument = await GetDocumentAsync(testsListPage);
+            var manualResults = testsListDocument.GetElementById("manual-results");
+
+            Assert.Contains("Smear", manualResults.TextContent);
+            Assert.Contains("Blood", manualResults.TextContent);
+            Assert.Contains("Negative", manualResults.TextContent);
+        }
+
+        [Fact]
+        public async Task PostEditOfManualTestResult_ReturnsSuccessAndAddsResultToTable()
+        {
+            // Arrange
+            var NotificationId = Utilities.DRAFT_ID;
+            var editUrl = GetCurrentPathForId(NotificationId) + TEST_ID;
+            var editPage = await Client.GetAsync(editUrl);
+            var editDocument = await GetDocumentAsync(editPage);
```

Comment:
Could we assert that the field that is about to be changed has its original value here? It goes slightly against the arrange/act/assert but I think it would be good to add an additional assert step